cmake/Modules/OpmStaticTargets.cmake

```diff
@@ -7,15 +7,22 @@
 
 # Macros
 function(opm_from_git repo name revision)
+  if(ARGN)
+    set(COMMANDS BUILD_COMMAND make ${ARGN}
```

Comment:
Are you sure this (`BUILD_COMMAND make ${ARGN}`) is safe?  What if I use, e.g., Ninja as my generator?

I would have thought that something along the lines of
```
cmake --build . -- ${ARGN} # --build DOT -- ${ARGN}
```
would be more general.  Then again, I'm not sufficiently well versed in the intricacies of recursive CMake builds to state this definitively.